FunctionalTableData/CollectionView/FunctionalCollectionData.swift

```diff
@@ -586,9 +586,11 @@ extension FunctionalCollectionData: UICollectionViewDelegate {
 	public func scrollViewDidScrollToTop(_ scrollView: UIScrollView) {
 		scrollViewDidScrollToTop?(scrollView)
 	}
-	
-	// MARK: - UIScrollViewAccessibilityDelegate
-	
+}
+
+// MARK: - UIScrollViewAccessibilityDelegate
+
+extension FunctionalCollectionData: UIScrollViewAccessibilityDelegate {
```

Comment:
how do these actually get called? Does UIScrollView internally check to see if its delegate conforms to this?

Reply:
Apparently so. `UIScrollView` tests if its delegate conforms to the protocol, and if it doesn't then it won't bother checking if the method itself has been implemented. Hence why the previous implementation (which was grouped in with the `UIScrollViewDelegaate` methods) wasn't being called.